app/WikiLifecycleEvents.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+// This class is supposed to contain information about certain events
+// in a wiki's lifecycle, e.g. the time of the last edit. Sources for these
+// points in time can be chosen as needed.
+class WikiLifecycleEvents extends Model
```

Comment:
Naming suggestions are welcome. I think the current one is ok, but maybe there's something better out there still.

Reply:
I think that naming already fits quite good :)